### PR TITLE
chore: local will now have its own secret file

### PR DESCRIPTION
**Pull request by dvince2**
# Description

Adding secrets to the local deploy as well.

Resolves #717

### CODE REVIEW

File: kubernetes/dev_deploy.sh [hunk 1]
```diff
@@ -8,6 +8,40 @@ HMI_SERVER_REPLACEMENT_FILE="overlays/dev/local/hmi/server/configmap/host.yaml"
...
+			ansible-vault encrypt --vault-id ~/askem-vault-id.txt "${FILE}"
+		done
+	done
+}
+
+restore() {
+	for SECRET_FILE in "${DECRYPTED_FILES[@]}"; do
+		git restore "${SECRET_FILE}"
+	done
+}
```
Comment (YohannParis) [inline kubernetes/dev_deploy.sh:43]: @bigglesandginger do you think this is possible to be refactored in its own script file at the root of `/kubernetes`?

**Review by YohannParis — APPROVED** [review]

File: kubernetes/dev_deploy.sh (hunk 1 above)
Comment (bigglesandginger) [reply kubernetes/dev_deploy.sh:43]: yes totally possible, I will do that

**Comment by bigglesandginger** [comment]
Added a `functions.sh` file to encapsulate the encryption and decryption functions

File: kubernetes/dev_deploy.sh (hunk 1 above)
Comment (YohannParis) [reply kubernetes/dev_deploy.sh]: Thanks, that was nice and easy!

**Pull request merged**